CHANGELOG.md

```diff
@@ -5,66 +5,110 @@ CHANGES:
 
 * auth/approle: Add maximum length of 4096 for approle role_names, as this value results in HMAC calculation [[GH-17768](https://github.com/hashicorp/vault/pull/17768)]
 * auth: Returns invalid credentials for ldap, userpass and approle when wrong credentials are provided for existent users.
-This will only be used internally for implementing user lockout. [[GH-17104](https://github.com/hashicorp/vault/pull/17104)]
-* core: Bump Go version to 1.19.2.
+  This will only be used internally for implementing user lockout. [[GH-17104](https://github.com/hashicorp/vault/pull/17104)]
+* core: Bump Go version to 1.19.3.
+* logging: Removed legacy environment variable for log format ('LOGXI_FORMAT'), should use 'VAULT_LOG_FORMAT' instead [[GH-17822](https://github.com/hashicorp/vault/pull/17822)]
+* plugins: Mounts can no longer be pinned to a specific _builtin_ version. Mounts previously pinned to a specific builtin version will now automatically upgrade to the latest builtin version, and may now be overridden if an unversioned plugin of the same name and type is registered. Mounts using plugin versions without `builtin` in their metadata remain unaffected. [[GH-18051](https://github.com/hashicorp/vault/pull/18051)]
 * plugins: `GET /database/config/:name` endpoint now returns an additional `plugin_version` field in the response data. [[GH-16982](https://github.com/hashicorp/vault/pull/16982)]
 * plugins: `GET /sys/auth/:path/tune` and `GET /sys/mounts/:path/tune` endpoints may now return an additional `plugin_version` field in the response data if set. [[GH-17167](https://github.com/hashicorp/vault/pull/17167)]
 * plugins: `GET` for `/sys/auth`, `/sys/auth/:path`, `/sys/mounts`, and `/sys/mounts/:path` paths now return additional `plugin_version`, `running_plugin_version` and `running_sha256` fields in the response data for each mount. [[GH-17167](https://github.com/hashicorp/vault/pull/17167)]
 * secrets/aws: do not create leases for non-renewable/non-revocable STS credentials to reduce storage calls [[GH-15869](https://github.com/hashicorp/vault/pull/15869)]
 * ui: Upgrade Ember to version 4.4.0 [[GH-17086](https://github.com/hashicorp/vault/pull/17086)]
-
 FEATURES:
 
-* core: Add user lockout field to config and configuring this for auth mount using auth tune to prevent brute forcing in auth methods [[GH-17338](https://github.com/hashicorp/vault/pull/17338)]
+* logging: Vault Agent supports logging to a specified file path via environment variable, CLI or config [[GH-17841](https://github.com/hashicorp/vault/pull/17841)]
```

Comment:
@calvn could you reach out to all the people on these linked PRs (I think it's mostly my team) and ask them to either move these out of the "Features" section or use the "Features" formatting? It looks like at least the logging ones might be more improvements. This isn't in the checklist but I try to keep on top of the reminders.

If people change the changelog files, this section will be automatically repaired when we re-gen the changelog, so for this PR it's not a problem to have them formatted like this.

Reply:
cc: @ncabatoff @peteski22 @hashishaw @hellobontempo on updating the format for these entries on the features section of the CL.

Reply:
I've raised a PR for this to resolve the log related PRs. It's set to auto merge when the build completes successfully. Thanks for pointing this out, as I learned something new today. :)

Reply:
This thread was lost in my email! Apologies - Opened PR #18182 to address

Reply:
@hellobontempo this change is part of the 1.13 preview and isn't urgent to address. As long as they're cleaned up by the next release we'll regenerate this part of the changelog (just the 1.13 preview portion), and they'll show up in the correct section then.

Reply:
ah thanks for clarifying! 😄